controllers/README.md

```diff
@@ -239,3 +239,77 @@ HTTP Response:
     "alarmEnd": string 
 }
 ```
+## Add Response Metric Log
+
+POST "/metrics/response"
+
+Request Body:
+
+```
+{
+    "userID": string, // MongoID
+    "alarmID": integer,
```

Comment:
How does the responder's app know the alarmID when the user he/she is responding to creates an alarm log (sent help request) ?

Reply:
Could the alarmID be sent as part of the help request?
Otherwise, I can make it get the alarmID itself by finding the latest alarm log from the user who needs help if the responder app knows their id/username

Reply:
We can attach the alarmId as part of the help request.